--- packages/renderer/src/strategies/DomStrategy.ts
+++ packages/renderer/src/strategies/DomStrategy.ts
@@ -179,15 +179,16 @@
           if (box) {
             const params = {
               screenshot: {
-                ...this.cdpScreenshotParams,
+                format: this.cdpScreenshotParams.format,
+                quality: this.cdpScreenshotParams.quality,
                 clip: { x: box.x, y: box.y, width: box.width, height: box.height, scale: 1 }
               },
               interval: this.frameInterval,
               frameTimeTicks: 10000 + frameTime
             };

-            return this.cdpSession!.send('HeadlessExperimental.beginFrame', params).then(({ screenshotData }: any) => {
-              if (screenshotData) {
-                const buffer = this.writeToBufferPool(screenshotData);
+            return this.cdpSession!.send('HeadlessExperimental.beginFrame', params).then((res: any) => {
+              if (res && res.screenshotData) {
+                const buffer = this.writeToBufferPool(res.screenshotData);
                 this.lastFrameBuffer = buffer;
                 return buffer;
               } else if (this.lastFrameBuffer) {
@@ -223,9 +224,9 @@
         frameTimeTicks: 10000 + frameTime
       };

-      return this.cdpSession.send('HeadlessExperimental.beginFrame', params).then(({ screenshotData }: any) => {
-        if (screenshotData) {
-          const buffer = this.writeToBufferPool(screenshotData);
+      return this.cdpSession.send('HeadlessExperimental.beginFrame', params).then((res: any) => {
+        if (res && res.screenshotData) {
+          const buffer = this.writeToBufferPool(res.screenshotData);
           this.lastFrameBuffer = buffer;
           return buffer;
         } else if (this.lastFrameBuffer) {
